client/Components/AssemblyAndDocuments.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'; 
+
+export default class AssemblyAndDocuments extends React.Component{
```

Comment:
I'm surprised everything is in a class component, but I don't know of any actual reasons why they shouldn't be, so carry on.